src/test/java/io/airlift/slice/AbstractSliceInputTest.java

```diff
@@ -247,6 +249,60 @@ public void verifyValue(SliceInput input, int valueIndex)
         });
     }
 
+    @Test
+    public void testReadDoubles()
+    {
+        class DoublesSliceInputTester
```

Comment:
We try to avoid method-local classes. It makes the code harder to follow. Pull it up one level.